sdk/python/approzium/authenticator.py

```diff
@@ -17,55 +18,66 @@
 import authenticator_pb2_grpc  # noqa: E402 isort:skip
 
 
-class Authenticator(object):
-    def __init__(self, address, iam_role=None):
-        self.address = address
+class AuthClient(object):
+    def __init__(self, server_address, iam_role=None):
+        self.server_address = server_address
         self.iam_role = iam_role
+        self.authenticated = False
+        self._counter = count(1)
+        self.n_conns = 0
 
+    @property
+    def attribution_info(self):
+        info = {}
+        info["authenticator_address"] = self.server_address
+        info["iam_role"] = self.iam_role
```

Comment:
Should we be logging this stuff at some point? I thought about it more and I finally _get it_ that the logs would be included with the client application. Do we already have any logging in the Python client? I'm not sure if I'm making a big ask or a small one.

Reply:
I believe It is up in the air, based on previous discussions. I think we reached that logging should be done by hooking into already existing logging frameworks. Right now, we provide the attribution_info to the user, if they want to "enrich" their logging info with it. At some point in the future, we would add the logging framework integration so that it is done automatically. We do have logging at the module-level, but I don't know if that is considered the same type of logging or not.